### PR TITLE
Update Developer Guide and User Guide - Tim

### PR DESCRIPTION
**Pull request by timleow**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -65,10 +65,22 @@ TaskHub is a **desktop app for managing contacts, optimized for use via a Comman
...
+Upon clicking help from the GUI as shown below,
+
+![circled_help_gui](images/circledHelpGUI.png)
+
+a message is displayed, with the link to our user guide. 
 
 ![help message](images/helpMessage.png)
 
+#### CLI Help
+Alternatively, quickly enter a `help` command into TaskHub to display to pop-up shown below.
```
Comment (rayyan35p) [inline docs/UserGuide.md:78]: Is it meant to be "...display _the_ pop-up shown below."?

**Review by rayyan35p — COMMENTED** [review]
Just a small error with the grammar if I understood what you were trying to say correctly. Should be good to merge after fixing that.

File: docs/UserGuide.md (hunk 1 above)
Comment (timleow) [reply docs/UserGuide.md:78]: yeah you're right, that's my bad

**Review by rayyan35p — APPROVED** [review]
LGTM!

**Pull request merged**